src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/jvm/JvmGenTuningPolicyTest.java

```diff
@@ -167,5 +176,19 @@ public void testEvaluate() {
     Assert.assertEquals(1, actions.size());
     Assert.assertTrue(actions.get(0) instanceof JvmGenAction);
     Assert.assertEquals(3, ((JvmGenAction) actions.get(0)).getTargetRatio());
+    mockCurrentRatio(5);
+    actions = policy.evaluate();
+    Assert.assertEquals(1, actions.size());
```

Comment:
We should avoid using multiple asserts separately in a single unit test. If the first one fails, the test would exit and the following asserts won't execute, leaving us with no information on whether the following asserts would also fail. Also I would suggest adding failure message string in the assertion statements as that would increase the readability of the code and would also help in future maintenance. This is true for all the assertions in the current test file.

Sample Statement for the added assertions: 

```
Assert.assertTrue("Reason as to what is the cause of this assertion failure", 
                1 == actions.size()
                && actions.get(0) instanceof JvmGenAction
                && 4 == ((JvmGenAction) actions.get(0)).getTargetRatio());
```